ui/src/app/projects/projects.component.ts

```diff
@@ -93,7 +94,13 @@ export class ProjectsComponent implements OnInit {
   }
 
   navigateJobs() {
-    let extras = {queryParams: {q: URLSearchParamsUtils.encodeURLSearchParams({extensions: {projectId: this.projectsControl.value}})}};
-    this.router.navigate(['jobs'], extras);
+    let jobsExtras = {queryParams: {q: URLSearchParamsUtils.encodeURLSearchParams({extensions: {projectId: this.projectsControl.value}})}};
+    let dashboardExtras = {queryParams: {projectId: this.projectsControl.value}};
```

Comment:
nit: I would prefer inlining these below or else defining the variables right before they are used.